test/release/examples/primers/learnChapelInYMinutes.chpl

```diff
@@ -38,43 +69,69 @@ myCplx = myInt + myImag ; // Another way to form complex numbers
 var myBool: bool = false; // Booleans
 var myStr: string = "Some string..."; // Strings
 
-// Some types can have sizes
+// Some types can have sizes.
 var my8Int: int(8) = 10; // 8 bit (one byte) sized int;
 var my64Real: real(64) = 1.516; // 64 bit (8 bytes) sized real
 
-// Typecasting
+// Typecasting.
 var intFromReal = myReal : int;
 var intFromReal2: int = myReal : int;
 
-// consts are constants, they cannot be changed after set in runtime.
+// Type aliasing.
+type chroma = int;        // Type of a single hue
+type RGBColor = 3*chroma; // Type representing a full color
+var black: RGBColor = ( 0,0,0 );
+var white: RGBColor = ( 255, 255, 255 );
+
+/*
+.. primers-yminutes-consts-params:
+
+Constants and Parameters
+------------------------
+*/
+
+// A ``const`` is a constant, and cannot be changed after set in runtime.
 const almostPi: real = 22.0/7.0;
 
-// params are constants whose value must be known statically at compile-time
-// Their value cannot be changed.
+// A ``param`` is a constant whose value must be known statically at compile-time.
+// Their value also cannot be changed.
 param compileTimeConst: int = 16;
 
-// The config modifier allows values to be set at the command line
-// and is much easier than the usual getOpts debacle
-// config vars and consts can be changed through the command line at run time
+// The ``config`` modifier allows values to be set at the command line.
+// Set with ``--VarName=Value`` or ``--VarName Value`` at runtime.
 config var varCmdLineArg: int = -123;
 config const constCmdLineArg: int = 777;
-// Set with --VarName=Value or --VarName Value at run time
 
-// config params can be set/changed at compile-time
+// ``config param``s can be set/changed at compile-time.
```

Comment:
I think this needs to be 
```
``config param`` s
```

or just drop the back ticks for config params here.